src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -108,12 +108,32 @@ struct sd_dhcp_client {
         usec_t start_delay;
 };
 
+/* RFC7844 section 3:
+   MAY contain the Parameter Request List option.
+   RFC7844 section 3.6:
+   The client intending to protect its privacy SHOULD only request a
+   minimal number of options in the PRL and SHOULD also randomly shuffle
+   the ordering of option codes in the PRL.  If this random ordering
+   cannot be implemented, the client MAY order the option codes in the
+   PRL by option code number (lowest to highest).
+*/
+/* NOTE: sending PRL options that Windows 10 RFC7844 implementation sends,
+ * ordered by code number, which in hexadecimal is:
+ * 37 0d 01 03 06 0f 1f 21 2b 2c 2e 2f 79 f9 fc */
```

Comment:
> I like the idea of hiding in the crowd ;)
:)
>  I don't get the comment though: 37 0d 01 ... is not the same as 1 3 6 ... below.
It is: 0x35 = 55 (PRL), 0x0d = 15 (option length), and " 01 03 06 0f 1f 21 2b 2c 2e 2f 79 f9 fc" the actual options, iirc. It was useful to me when checking the tcpdump hex dump.
I think i should remove that comment, or use that hex string in a test, but i'm not sure how to create the test with it.